vcf2fhir/common.py

```diff
@@ -135,13 +175,76 @@ def validate_has_tabix(has_tabix):
     return True
 
 
+def get_codeable_concept(system, code, display=None):
+    codeable_concept = {"coding": [{}]}
+    codeable_concept['coding'][0]['system'] = system
+    codeable_concept['coding'][0]['code'] = code
+    if display is not None:
+        codeable_concept['coding'][0]['display'] = display
+    return concept.CodeableConcept(codeable_concept)
+
+
+# def validate_alt_simple(alt):
+#     alt_pattern_simple =\
+#         re.compile(
+#             r"^([a-zA-Z]+)(,[a-zA-Z]+)*$|" +
+#             # Match character string and comma separated
+#             # list of character strings
+#             r"^\.$"
+#             # Match '.'
+#         )
+#     alt = ",".join(list(map(str, list(alt))))
+#     if alt_pattern_simple.match(alt):
+#         return True
+#     else:
+#         return False
+
+
+# def validate_alt_sv(alt):
+#     alt_pattern_sv =\
+#         re.compile(
+#             r"^([a-zA-Z]+)(,[a-zA-Z]+)*$|" +
+#             # Match character string and comma separated
+#             # list of character strings
+#             r"^\.$|" +
+#             # Match '.'
+#             r"^(<[A-Z*\d*]+>)(,<[A-Z*\d*]+>)*$"
```

Comment:
(multiple occurrences)
Do we need commented code?

Reply:
Hi @srgothi92, I have removed all the commented code from this PR. I have stored a copy of that code on my local computer if it is required anytime in the future.